zkevm-circuits/src/super_circuit.rs

```diff
@@ -407,19 +401,12 @@ impl<F: Field> Circuit<F> for SuperCircuit<F> {
             max_withdrawals: self.circuits_params.max_withdrawals,
             max_calldata: self.circuits_params.max_calldata,
             mock_randomness: self.mock_randomness,
+            feature_config: self.feature_config,
         }
     }
 
     fn configure_with_params(meta: &mut ConstraintSystem<F>, params: Self::Params) -> Self::Config {
-        Self::Config::new(
-            meta,
-            SuperCircuitConfigArgs {
-                max_txs: params.max_txs,
-                max_withdrawals: params.max_withdrawals,
-                max_calldata: params.max_calldata,
-                mock_randomness: params.mock_randomness,
-            },
-        )
+        Self::Config::new(meta, params)
     }
 
     fn configure(_meta: &mut ConstraintSystem<F>) -> Self::Config {
```

Comment:
Note: Previously I suggested doing `unreachable!()` in the `configure` of the `EvmCircuit`, it's following this pattern.